docs/site/Debugging-with-VSC.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Can we rename this file to be `Debugging-with-vscode.md` to be consistent with the html page?